DXMainClient/DXGUI/Multiplayer/GameListBox.cs

```diff
@@ -185,6 +194,24 @@ public override void Initialize()
                 ClientConfiguration.Instance.HoverOnGameColor);
 
             loadedGameTextWidth = (int)Renderer.GetTextDimensions(LOADED_GAME_TEXT, FontIndex).X;
+
+            InitGameDifficultyIcons();
+        }
+
+        private void InitGameDifficultyIcons()
+        {
+            for (int i = 0; i < OnlineGameDifficultyOptions.Length; i++)
+            {
+                Texture2D gd = AssetLoader.LoadTexture("game_difficulty_" + i + ".png");
```

Comment:
We use either camelCase or PascalCase for file names by default IIRC, please verify.

Also use string interpolation, makes it more readable.